_posts/2017-10-23-pair-programming-with-perry.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+authors:
+    - justinSchoen
+date: 2017-10-23 3:00:00 -0500
+title: Pair Programming with Perry
+---
+
+Hey, everyone. Today I’m going to interview Perry, a pair programmer practitioner, about pair programming, his experiences, and he’ll give us some tips on pairing effectively.
+
+<hr/>
+<span class='os-green'>*So, Perry, tell me, why should anyone pair program?*</span>
+<div style='margin-left: 2em;' >
+    Great question, you’ve earned a schrute buck for your curiosity, and yes, those are transferrable for Stanley nickels. Currency aside, here’s a common scenario: you can’t get unstuck on a problem so you seek someone’s help and in explaining the problem to them the solution comes to you, sometimes the other person doesn’t even say anything to help. This isn’t a new concept and it’s the reason you’ll sometimes hear people talk about rubber-duck programming, which is when you explain your code line by line to a rubber duck with intentions of discovering bugs and confusing logic, or getting unstuck.
+</div>
+<br>
+<span class='os-green'>*It sounds to me like anytime I get stuck I should just start talking out loud to inanimate objects. Is that correct?*</span>
+<div style='margin-left: 2em;' >
+    Well, not really. Though that can be useful, wouldn’t you rather just get stuck far less often if you could?
```

Comment:
This makes it sound like the main benefit to pair programming is not getting stuck as often, but there are a lot of other reasons. I might combine this and a few of the other shorter exchanges below into a larger, more pointed paragraph about the benefits.

Reply:
Yeah I don't love my first two paragraphs. I start out asking why anyone should pair and my answer just involves getting unstuck. Needs to be reworked